plugin/trino-ranger/conf/ranger-policymgr-ssl.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
```

Comment:
How these files are used?

Reply:
@dprophet ^^ I assume these files are default confs? If so, we should likely make a tutorial on these. I can also help with this. cc: @mosabua

Reply:
These files are pure example files. I didnt know what/where to put them.

Reply:
Yeah, perhaps we could add these for use with a test and later consider making a tutorial with them.